base_app/src/app/app-tab-overlay/overlay-page.ts

```diff
@@ -619,6 +655,28 @@ export class OverlayPageComponent {
                 return value;
             }
 
+            /******************************************************************/
+            /* Start of section lost with checkin on 9/10.2020 2:07 PM change
+               Not sure if this code is needed, but without it, "if (fmt === '')"
+               (used below this block) will always evaluate to true.
+             */
+            if (isImplemented) {
+                css = this.getImpOverlayCssDef(varName);
+                if (css.length > 0) {
+                    let cmd = 'var e=document.getElementById("' + varName + '");' +
+                        'var e_val=document.getElementById("' + varName + '_value");' +
+                        'if (e) {e.style="' + css + '";}';
+
+                    const val_css = this.getImpOverlayCssDef(varName + '_value');
+                    if (val_css.length > 0) {
+                        cmd += 'if (e_val) {e_val.style="' + val_css + '";}';
+                    }
+                    setTimeout(cmd, 50);
```

Comment:
Probably change the timeout to 5 (or 1) instead of 50, since I think it only needs to be executed outside the current call stack.  Don't think there's any reason to wait 50ms